src/Sylius/Behat/Page/Shop/Checkout/SelectPaymentPage.php

```diff
@@ -32,6 +33,21 @@ public function getRouteName()
      */
     public function selectPaymentMethod($paymentMethod)
     {
+        $driver = $this->getDriver();
+        if ($driver instanceof Selenium2Driver) {
+            $driver->executeScript("
+                (function ( $ ) {
+                    'use strict';
+                
+                    $(document).ready(function() {
+                        $('#sylius_checkout_select_payment_payments_0_method_0').parent().trigger('click');
```

Comment:
@Zales0123 no, it's semantic-ui's element and selenium cannot click onto it. 
@pamil I think, that I can pass id (or other selector) based on $paymentMethod argument.